test/integration/puppetlabs/services/jruby/jruby_puppet_pool_int_test.clj

```diff
@@ -450,6 +450,9 @@
             jruby-scripting-container (:scripting-container jruby-instance)
             jruby-env (.runScriptlet jruby-scripting-container "ENV")]
         (try
+         ;; Note that if this test is run in an environment where the
+         ;; variables HTTP_PROXY, HTTPS_PROXY, NO_PROXY, or their lowercase
+         ;; variants are set, this will fail.
           (is (= #{"HOME" "PATH" "GEM_HOME" "GEM_PATH"
```

Comment:
I think we still want to fail if other environment variables that we're not explicitly intending to filter out end up get passed through, though.  The equality comparison is still useful.  I just think it would be good to ignore the http proxy-related ones since they may or may not exist in whatever environment the tests are run from.